components/event-header.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="event-header">
+    <h1 class="event-header__name">
+      <span class="event-header__location">{{ location }}</span>
+      {{ name }}
+    </h1>
+    <nav class="language-switch">
+      <ul class="language-switch__list">
+        <li v-for="{ locale } in _allNameLocales" :key="locale">
```

Comment:
I don't know if it's possible, but they wanted the Chinese language option to be the first option, and other languages the second, third, ... option.

Reply:
A more generic approach would work, sort the locales so that the English language is first. It's more logic though, do you think this is needed @tiaravanni?

Reply:
yes unfortunately it is.. ole explicitly mentioned in the event page, Chinese should be the default. Can you choose default language in dato, may be?

Reply:
@tiaravanni this is not about the default language though. Just about the order at the top of the page:
![image](https://user-images.githubusercontent.com/5701149/84507849-bbf0af80-acc1-11ea-80cf-ebd2b333051d.png)

English and then Chinese or the other way around.

Reply:
it doesnt matter, if english is on the right

Reply:
It should be Chinese first, then English

Reply:
or left